rmf_demos_tasks/rmf_demos_tasks/teleop_action.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
```

Comment:
Let's rename this file to `teleop_robot.py`. The `teleop_action` is a bit ambiguous given we have `dispatch_teleop` and `dispatch_action` scripts.